lib/auth/grpcserver.go

```diff
@@ -377,6 +377,23 @@ func (g *GRPCServer) Ping(ctx context.Context, req *proto.PingRequest) (*proto.P
 	return &rsp, nil
 }
 
+// CreateUser inserts a new entry in a backend.
+func (g *GRPCServer) CreateUser(ctx context.Context, req *services.UserV2) (*empty.Empty, error) {
+	auth, err := g.authenticate(ctx)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+
+	err = auth.CreateUser(ctx, req)
+	if err != nil {
+		return nil, trail.ToGRPC(err)
+	}
+
+	log.Infof("%v user created", req.GetName())
```

Comment:
prefer using `%q` to print non-constant strings.
it'll catch weird whitespace and other invisible characters.

Reply:
revised